RecoHGCal/TICL/python/TrkEMStep_cff.py

```diff
@@ -59,11 +63,13 @@
     seeding_regions = "ticlSeedingTrkHFNose",
     time_layerclusters = "hgcalLayerClustersHFNose:timeLayerCluster",
     itername = "TrkEMn",
+    # eid_graph_path: might as well have a separately trained model for nose
     pluginPatternRecognitionByCA = dict(
         filter_on_categories = [0, 1],
         min_layers_per_trackster = 5,
         pid_threshold = 0.,
         shower_start_max_layer = 5 #inclusive
+        # eid_n_layers = 8
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant.

(the default is apparently 50)